lib/config/fileconf.go

```diff
@@ -854,7 +854,9 @@ func (a *AuthenticationConfig) Parse() (types.AuthPreference, error) {
 }
 
 type UniversalSecondFactor struct {
-	AppID                string   `yaml:"app_id"`
+	AppID string `yaml:"app_id"`
+	// Facets kept only to avoid breakages during Teleport updates.
```

Comment:
nit
```suggestion
	// Deprecated: Facets kept only to avoid breakages during Teleport updates.
```

Reply:
Done (comment is slightly different, but added "Deprecated: " at the start).